### ISSUE TITLE
Custom Dataviewer editor with prohibition symbol

### ISSUE THREAD

**Issue by gepponline** [thunkable/thunkable-issues#885]
I'm trying to edit a custom Dataviewer list layout but when i click on EDIT it opens an editor where I can select object, but if i try to change any properties, for example dimension or alignment or padding or whatever else, a prohibition symbol appear at the mouse pointer.
Why?
![1619707170419](https://user-images.githubusercontent.com/52273816/116569487-ba47ce80-a909-11eb-813d-9058489b754a.jpg)

**Comment by jane-d-alt** [comment]
Hi there,

Do you see this behavior when trying to edit your layout from the My Data Viewer Layouts tab?

**Comment by gepponline** [comment]
Yes

Il lun 17 mag 2021, 12:39 jane-d-alt ***@***.***> ha scritto:

**Comment by gepponline** [comment]
Yes, I save my custom layout (I've tried both column or entire screen) then
I open my Data Viewer layout Tab, and click on edit link.
Thank you

Il lun 17 mag 2021, 12:39 jane-d-alt ***@***.***> ha scritto:
